### DISCUSSION TITLE
Powershell formatting does not work like expected / configured

### DISCUSSION THREAD

**Discussion by haukebruno** [mason-org/mason.nvim#1436] [Q&A] [answered]
Hey all,

I use LazyVim (latest) which has Mason (main branch, commit fe9e34a) configured right away. 

In powershell files the language server is active, but while saving the file all curly brackets move to a new line. The setting `powershell.codeFormatting.openBraceOnSameLine` is `true` for me, so I would assume that the backets should stay in their location (right after a function definition for instance).

How can I do further debugging on this? I'm quite new to Mason, so I guess I'm just missing something.

Have a nice one,
Hauke

**Comment by williamboman** [comment, marked as answer]
Hello! Mason is not involved in configuring or starting LSP servers, so your best bet is to investigate which configurations are applied to the server and if it aligns with how the upstream LSP server expects them to be provided.

**Reply by haukebruno** [reply]
Thanks for clarifying :)
